cea/interfaces/dashboard/api/project.py

```diff
@@ -22,92 +22,89 @@
 
 PROJECT_PATH_MODEL = api.model('Project Path', {
     'path': fields.String(description='Path of Project'),
-    'scenario': fields.String(description='Path of Scenario')
 })
 
-PROJECT_MODEL = api.inherit('Project', PROJECT_PATH_MODEL, {
-    'name': fields.String(description='Name of Project'),
-    'scenario': fields.String(description='Name of Current Scenario'),
-    'scenarios': fields.List(fields.String, description='Name of Current Scenario')
+SCENARIO_PATH_MODEL = api.inherit('Scenario Path', PROJECT_PATH_MODEL, {
+    'scenario': fields.String(description='Name of Scenario')
 })
 
-NEW_PROJECT_MODEL = api.model('New Project', {
+PROJECT_MODEL = api.inherit('Project', SCENARIO_PATH_MODEL, {
     'name': fields.String(description='Name of Project'),
-    'path': fields.String(description='Path of Project')
+    'scenarios': fields.List(fields.String, description='Name of Current Scenario')
 })
 
 
 @api.route('/')
 class Project(Resource):
     @api.marshal_with(PROJECT_MODEL)
+    @api.doc(params={'path': 'Path of Project (Leave blank to use path in config)'})
     def get(self):
-        config = current_app.cea_config
-        name = os.path.basename(config.project)
-        if not os.path.exists(config.project):
-            abort(400, 'Project path does not exist')
-        if not os.path.exists(config.scenario):
-            config.scenario_name = ''
-            config.save()
-        return {'name': name, 'path': config.project, 'scenario': config.scenario_name,
-                'scenarios': list_scenario_names_for_project(config)}
-
-    @api.doc(body=PROJECT_PATH_MODEL, responses={200: 'Success', 400: 'Invalid Path given'})
-    def put(self):
-        """Update Project"""
-        config = current_app.cea_config
-        payload = api.payload
-
-        if 'path' in payload:
-            path = payload['path']
-            if os.path.exists(path):
-                config.project = path
-                if 'scenario' not in payload:
-                    config.scenario_name = ''
-                    config.save()
-                    return {'message': 'Project path changed'}
-            else:
-                abort(400, 'Path given does not exist')
+        project_path = request.args.get('path')
+        if project_path is None:
+            config = current_app.cea_config
+            scenario = config.scenario_name
```

Comment:
I think it's a bit confusing to use the word `scenario` when referring to the `general:scenario-name` parameter - let's rename this to `scenario_name`. That way, `scenario` is always used for the string ` "{general:project}:{general:scenario-name}"`